src/components/tl/ucp/tl_ucp.h

```diff
@@ -69,15 +70,10 @@ typedef struct ucc_tl_ucp_task ucc_tl_ucp_task_t;
 typedef struct ucc_tl_ucp_team {
     ucc_tl_team_t              super;
     ucc_status_t               status;
-    int                        context_ep_storage; /*< The flag
-              indicates whether ucp endpoints are stored on the
-              ucc_tl_ucp_context or are they created per-team.
-              This optimization is only possible when user provides
-              the necessary rank mappings team_rank->context_rank. */
-    ucp_ep_h                  *eps;
     ucc_rank_t                 size;
     ucc_rank_t                 rank;
     ucc_tl_ucp_addr_storage_t *addr_storage;
+    int                        ctx_addr_exchange;
```

Comment:
what is the meaning of ctx_addr_exchange? is it used?

Reply:
It was old flow (potentially eps could be stored on CTX) - removed that now. Maybe we will still implement exchange of addresses during ctx create (in global mode) but later.